ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/ClientOptionsImpl.java

```diff
@@ -238,6 +249,11 @@ public int getHttp2MultiplexingLimit() {
     return http2MultiplexingLimit;
   }
 
+  @Override
+  public Map<String, String> getRequestHeaders() {
+    return requestHeaders == null ? null : new HashMap<>(requestHeaders);
```

Comment:
Would it be better to return an empty map?

Reply:
I don't feel strongly, though it is a public API so we should decide now. (The javadoc I added says it returns null if it was never set but that can easily be changed.) Would you expect an empty map? I can make that change in order to be consistent with the String fields returning the empty string if not set, rather than null.

Reply:
Hmm... I tried to look for other similar API instances briefly. 

I did this: https://github.com/confluentinc/ksql/blob/master/ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/HttpRequestImpl.java#L29-L30 where empty maps are used rather than null values.  I think that's a slight node to changing to empty maps.